### ISSUE TITLE
Latency/lag in GUI (scrttv)

### ISSUE THREAD

**Issue by FMassin** [seiscomp-macOS/seiscomp#2]
Hello

I have noticed rather extreme lag times while using the GUI with casual datasets. For example, using scrttv is much slower than on linux with the same mseed data.

Is there a way to improve the performance of the rendering in GUIs?

Cheers

Fred

**Comment by gilcel** [comment]
Sorry for the long delay. Which macOS version and Platform (Intel or Silicon) ?
Strange since we have Mac mini's here from 2013 (INTEL) with about > 20 stations and no lag at all with scrttv, scmv and other apps open.

**Comment by a-oth** [comment]
Hi Fred, 
Adding to Gilles' response, my experience is overall without problems with the GUIs, except indeed with scrttv and the picker window in scolv, which is related to screen resolution (HiDPI). For instance, on my iMac 5K, if I stay with HiDPI resolution, then scrttv or the picker will hang easily and become unresponsive. If I use for instance the software SwitchResX for reducing the resolution, then scrttv and the picker will work fine, just like the other GUIs.

**Comment by FMassin** [comment]
Ok, Thanks! So that is my retina laptop screen and/or my desktop hidpi 2nd screen. I will do more tests with poorer screens 😆 But there must be other unix users with hidpi screen resolution in the linux-based seiscomp community, I wonder if the seiscomp forum would have heard about this issue ...

**Comment by gilcel** [comment]
Since SeisComP GUI (windows etc.) is based on Qt5.1x you can try the following environment variables described here:
https://doc.qt.io/qt-5/highdpi.html

First stop all your SeisComP GUI applications then execute the following lines in your Terminal.app:
```
export QT_AUTO_SCREEN_SCALE_FACTOR=1
export QT_ENABLE_HIGHDPI_SCALING=1
```

Restart your SeisComP apps...

Some good advices here in chapter 5: GUI Toolkits  **Qt 5**:
https://wiki.archlinux.org/title/HiDPI

**Comment by FMassin** [comment]
Ho I think that helps ! But it is still a bit choppy ...

**Comment by gilcel** [comment]
Yeah we made some tests with scrttv and indeed it's choppy with HiDpi turned ON...
I will try to disable it in the source code of scrttv and do some tests...will let you know.

**Comment by FMassin** [comment]
Great!
